jcl/src/java.base/share/classes/java/lang/invoke/MemberName.java

```diff
@@ -72,7 +80,11 @@ String getMethodDescriptor() {
 	}
 	/*[IF Java11]*/
 	public boolean isFinal() {
-		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
+		if (mh instanceof FieldHandle) {
+			return ((FieldHandle)mh).isFinal();
+		}
+		/*[MSG "K0675", "Unexpected MethodHandle instance"]*/
+		throw new Error(com.ibm.oti.util.Msg.getString("K0675")); //$NON-NLS-1$
```

Comment:
This should throw `InternalError`.  Can you expand on the error message as well?  It will make it clear what went wrong

Reply:
Fixed, also expanded the error message to include the `mh`. 
@DanHeidinga please have another look.